src/sentry/plugins/helpers.py

```diff
@@ -26,7 +26,7 @@ def reset_options(prefix, project=None, user=None):
 
 def set_option(key, value, project=None, user=None):
     if user:
-        result = UserOption.objects.set_value(user, project, key, value)
+        result = UserOption.objects.set_value(user, key, value, project)
```

Comment:
I feel like I wouldn't want to use arg order for this anymore and we should explicitly say `project=project`.